src/Responder.php

```diff
@@ -46,20 +46,20 @@ public function __construct(SuccessResponseBuilder $successResponse, ErrorRespon
     /**
      * Generate an error JSON response.
      *
-     * @param  string|null $errorCode
+     * @param  mixed|null $errorCode
      * @param  int|null    $statusCode
      * @param  mixed       $message
      * @return \Illuminate\Http\JsonResponse
      */
-    public function error(string $errorCode = null, int $statusCode = null, $message = null):JsonResponse
+    public function error($errorCode = null, int $statusCode = null, $message = null):JsonResponse
     {
         if ($exception = config("responder.exceptions.$errorCode")) {
             if (class_exists($exception)) {
                 throw new $exception();
             }
         }
 
-        return $this->errorResponse->setError($errorCode, $message)->respond($statusCode);
+        return $this->errorResponse->setError($errorCode, $message)->respond($statusCode, [], false);
```

Comment:
Why are we overriding the config value for error responses here? I feel like if you enable the success flag you should get it for both success and error responses.